cves/CVE-2012-6617.yml

```diff
@@ -296,8 +294,8 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
-    note:
+    applies: true
+    note: We need to verify that all vectors coming into the system are read correctly and will not cause a denial of service.
```

Comment:
This is also a distrust inputs issue because they cannot trust that strings in rtp format will be safe.

Reply:
I agree with this ^

Reply:
Updated